include/gatekeeper_net.h

```diff
@@ -298,6 +308,11 @@ void lua_free_iface(struct gatekeeper_if *iface);
 
 int ethertype_filter_add(uint8_t port_id, uint16_t ether_type,
 	uint16_t queue_id);
+
+/*
+ * TODO Whenever there is a functional block that calls this function,
+ * one needs to deal with the variable IP headers, e.g., the GK-GT unit.
+ */
```

Comment:
Although, I suppose it's possible to make the assumption that the GK-GT Unit never receives packets with a variable-length IP header. @AltraMayor, is this an assumption the code should make?